src/App.jsx

```diff
@@ -2,63 +2,146 @@ import './App.css'
 import SectionContentWrapper from './components/sectionContentWrapper/SectionContentWrapper.jsx';
 import novilogo from './assets/novi-logo-color-transparent.png';
 import Navbar from './components/navbar/Navbar.jsx';
+import InformationCard from "./components/informationCard/InformationCard.jsx";
 
 function App() {
 
-	return (
-		<>
-			<header className="header">
-				<div className="header-content">
-					<Navbar/>
-					<h1>Dé hogeschool voor ICT</h1>
-				</div>
-			</header>
-			<main>
-				<section className="courses-section">
-					<SectionContentWrapper>
-						<h2>Onze hbo ICT-opleidingen</h2>
-						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut illum impedit inventore iste
-							nostrum omnis quae, quam quis rem ut.</p>
-					</SectionContentWrapper>
+    return (
+        <>
+            <header className="header">
+                <div className="header-content">
+                    <Navbar/>
+                    <h1>Dé hogeschool voor ICT</h1>
+                </div>
+            </header>
+            <main>
+                <section className="courses-section">
+                    <SectionContentWrapper>
+                        <h2>Onze hbo ICT-opleidingen</h2>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut illum impedit inventore iste
+                            nostrum omnis quae, quam quis rem ut.</p>
 
-					<SectionContentWrapper>
-						<h2>ICT-Bootcamps</h2>
-						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut illum impedit inventore iste
-							nostrum omnis quae, quam quis rem ut.</p>
-					</SectionContentWrapper>
-					<SectionContentWrapper>
-						<h2>ICT-Cursussen</h2>
-						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut illum impedit inventore iste
-							nostrum omnis quae, quam quis rem ut.</p>
-					</SectionContentWrapper>
-				</section>
-				<section className="testimonial-section">
-					<SectionContentWrapper>
-						<h2>Voor wie?</h2>
-						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut illum impedit inventore iste
-							nostrum omnis quae, quam quis rem ut.</p>
-					</SectionContentWrapper>
-				</section>
-				<section className="talentpool-section">
-					<SectionContentWrapper>
-						<h2>NOVI&apos;s Talent Pool</h2>
-						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut illum impedit inventore iste
-							nostrum omnis quae, quam quis rem ut.</p>
-					</SectionContentWrapper>
-				</section>
-				<section className="news-section">
-					<SectionContentWrapper>
-						<h2>Meer NOVI?</h2>
-						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut illum impedit inventore iste
-							nostrum omnis quae, quam quis rem ut.</p>
-					</SectionContentWrapper>
-				</section>
-			</main>
-			<footer className="footer-section">
+                        <div className="info-card-wrapper">
```

Comment:
Deze class zou ik misschien eerder `info-cards-container` noemen, omdat hij om allemaal kleinere elementen heen staat. Over het algemeen: _"In programming languages the word container is generally used for structures that can contain more than one element. A wrapper instead is something that wraps around a single object to provide more functionalities and interfaces to it."_